packages/gasket-utils/lib/try-resolve.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Tries to require.resolve a module, but ignores if it is not found.
+ * If not found, result will be null.
+ *
+ * @example
+ * const { tryResolve } = require('@gasket/utils');
+ *
+ *  let modulePath: string = tryResolve('../might/be/a/path/to/some/file');
+ *
+ *  if(modulePath) {
+ *   modulePath = require(modulePath)
+ * }
+ *
+ * @param {string} modulePath - Module to import
+ * @param {string[]} paths - Paths to search for the module
+ * @returns {string} module path
+ */
+function tryResolve(modulePath, paths = []) {
+  try {
+    return require.resolve(modulePath, { paths });
```

Comment:
Just match this helper's signature up to [resolve.require](https://nodejs.org/docs/latest-v16.x/api/modules.html#requireresolverequest-options)
```suggestion
function tryResolve(modulePath, options) {
  try {
    return require.resolve(modulePath, options);
```